### ISSUE TITLE
RaaS: Support linking account between two social networks (TFS #36345)

### ISSUE THREAD

**Issue by shirlyleshed** [gigya/wordpress#25]
The following feature was introduced in Gigya v5.2:
User Story 18227:ScreenSet plugin - Allow for linking social accounts (to each other) via RaaS
## Goal:

Make sure that the module should work properly in this scenario.
I.e. - the module shouldn't delete second account in Gigya and shouldn't present error message to the user.
At the end of the process, the user should have one account in Gigya and one account in Drupal with both social identities linked to that account.
## Implementation:

Basically the default screen-sets in the Drupal-RaaS admin needs to be updated to the new Gigya default screen-sets. Here are the list of changes (from old => to new):

Login-web => Default-RegistrationLogin
Mobile-login => DefaultMobile-RegistrationLogin
Profile-web => Default-ProfileUpdate
Mobile-profile => DefaultMobile-ProfileUpdate

See also the new default screen-sets here:
http://developers.gigya.com/010_Developer_Guide/10_UM360/040_Raas/030_Markup/010_Default_Screen_Sets
